amulet/api/format_loader.py

```diff
@@ -48,11 +48,18 @@ def _find_formats(search_directory: str = None):
             )
             continue
 
-        modu = importlib.import_module(
```

Comment:
What's the difference between the old version and this one? This method seems more confusing, especially the usage of utils

Reply:
I agree it's a little more confusing, but it behaves better when in an actual module/package. It's also used in the importlib module examples: https://docs.python.org/3.7/library/importlib.html#importing-a-source-file-directly